Standards/scs-XXXX-v1-volume-backup-service.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Volume Backup Functionality
+type: Standard
+status: Proposal
+track: IaaS
+---
+
+## Introduction
+
+OpenStack offers a variety of resources where users are able to transfer and store data in the infrastructure.
+A prime example of these resources are volumes which are attached to virtual machines as virtual block storage devices.
+As such they carry potentially large amounts of user data which is constantly changing at runtime.
+It is important for users to have the ability to create backups of this data in a reliable and effifcient manner.
+
+## Terminology
+
+| Term | Meaning |
+|---|---|
+| CSP | Cloud Service Provider, provider managing the OpenStack infrastructure |
```

Comment:
Images and Volumes are referred throughout this document. Maybe they should be also explained here, so people always can look up the differences.

Reply:
Good point. Added.